src/common/token/arweave.ts

```diff
@@ -54,7 +54,7 @@ export class ArweaveToken implements TokenTools {
 
     this.arweave =
       arweave ??
-      Arweave.init({
```

Comment:
@fedellen - the only SDK change, i think it's just how bundlers tree shake the arweave import here and they use `node` by default - causing the `default.init()` not to exist in web projects. more to understand there but worked with my local example so will ask others to test.